lib/content.js

```diff
@@ -27,6 +27,17 @@ function getMediaTypeFromHttpReq( httpReq ) {
     return null;
 }
 
+function getPreferredStatusFromHttpReq( httpReq ) {
+    var matchedStatus;
+    if ( 'prefer' in httpReq.headers ) {
```

Comment:
Good shout. I've factored out the part that retrieves the header, and made the function generic so it can be used to extract the parameter from a specified preference assigned to a prefer header. 

It is not yet capable of parsing any prefer header, only those that take the form `Prefer: foo=bar`, but I think supporting any possible prefer header syntax is outside the scope of this change!

http://tools.ietf.org/html/rfc7240#section-1.1